kubespawner/clients.py

```diff
@@ -28,6 +29,8 @@ def shared_client(ClientType, *args, **kwargs):
     so that concurrent calls to shared_client
     will all return the same instance until
     all references to the client are cleared.
+
+    You should usually await load_config before calling this.
```

Comment:
I look for more clarity. I'm working under the idea that "you must" rather than "you should". Do you agree @athornton?

I'm looking through the code, aiming to enforce that - which we need to do - right?

Reply:
I agree.